Code/Framework/AzToolsFramework/Tests/ActionManager/ActionManagerTests.cpp

```diff
@@ -107,6 +107,108 @@ namespace UnitTest
         EXPECT_TRUE(actionTriggered);
     }
 
+    TEST_F(ActionManagerFixture, GetActionName)
+    {
+        m_actionManagerInterface->RegisterActionContext("", "o3de.context.test", {}, m_widget);
+        
+        AzToolsFramework::ActionProperties actionProperties;
+        actionProperties.m_name = "Test Name";
+        m_actionManagerInterface->RegisterAction("o3de.context.test", "o3de.action.test", actionProperties, []{});
+
+        auto outcome = m_actionManagerInterface->GetActionName("o3de.action.test");
+        EXPECT_TRUE(outcome.IsSuccess());
+        EXPECT_EQ(outcome.GetValue(), "Test Name");
```

Comment:
suggestion: You could use a string matcher for these test (they should give more useful output when they fail) - See this link - http://google.github.io/googletest/reference/matchers.html#string-matchers

e.g. `EXPECT_THAT(outcome.GetValue(), ::testing::StrEq("Test Name"));` (I think that should work 😝)